README.md

```diff
@@ -96,15 +113,11 @@ This principle of being unopinionated manifests in several important ways includ
 * A flat component architecture that lets you compose what you need without struggling with rigid patterns and complex objects.
 * Separating base components from styles and design systems to support multiple implementations without re-writing or duplicating styles. Use the design system to customize existing styled components, or build your own styles, with your design system, without having to rebuild or duplicate the base components.
 * Framework agnostic tooling that lets you use our development tools with any view framework.
-* The ability to replace almost any FAST package with your package of choice. Get started with our animation package and add more as you need them. Alternatively, use our suite of packages to build your next project from the ground up; it's your call.
-
-### UI development and style guide tools
-
-When developing components and views, excellent development tooling can make all the difference. FAST offers development and style guide tools that work with FAST components, components from other frameworks, or your components.
+* The ability to replace almost any FAST package with your package of choice. Just use one package or leverage our entire suite of packages to build your next project from the ground up; it's your call.
 
-Try out component properties with an auto-generated property UI, get a live preview of the code based on any property’s configuration, preview localization (RTL/LTR) and themes, and preview component compositing with a transparency grid.
+### Tooling
 
-Also, we built FAST development tools from re-usable packages, so if you have special needs, you can build your tools from the same shared libraries.
+Leverage our (upcoming!) FAST CLI to accelerate your design or development workflow. You can create new projects from scratch, create design systems, and add or customize from dozens of components in our foundation library.
```

Comment:
Do we think this is premature? I'd like to add this once beta is ready.

Reply:
I wasn't sure. This whole section of the docs seems a bit odd without some more content though.